pyswervedrive/chassis.py

```diff
@@ -36,11 +36,17 @@ def __init__(self):
         self.field_oriented = False
         self.momentum = False
         self.automation_running = False
+        self.acceleration = 2
+        self.deceleration = -0.25
```

Comment:
These appear to be unused.

Reply:
They are used in the autonomous for trajectory generation

Reply:
In the state of the tree at this commit, these are unused.